src/createWidgetBase.ts

```diff
@@ -143,6 +149,24 @@ function manageDetachedChildren(instance: Widget<WidgetProperties>): void {
 	});
 }
 
+function bindFunctionProperties(instance: Widget<WidgetProperties>, properties: WidgetProperties) {
+	Object.keys(properties).forEach((propertyKey) => {
+		const func = properties[propertyKey];
+		const bind = properties.bind;
+
+		if (typeof func === 'function') {
```

Comment:
or are you talking of the block? if so its specific to properties in `widget-core`, so wouldn't be applicable in `dojo/core`